packages/frontend/src/views/account/my-tickets.vue

```diff
@@ -35,8 +35,19 @@
   <input type="checkbox" id="transfer-ownership-modal" class="modal-toggle">
   <label for="transfer-ownership-modal" class="cursor-pointer modal" style="background-color: #00000055;">
     <label class="relative modal-box" for="">
-      <h3 class="text-lg font-bold">Congratulations random Interner user!</h3>
-      <p class="py-4">You've been selected for a chance to get one year of subscription to use Wikipedia for free!</p>
+      <h3 class="text-lg font-bold">Enter Recipient Username To Transfer</h3>
+      <div class="grid grid-cols-1 gap-6 mt-4 ">
+        <div>
+          <label class="text-header" for="Name">username</label>
+          <input id="edit-user-settings-username-input" type="username" class="box-text">
```

Comment:
ถ้า id ของ input เป็นตามนี้ for ของ label ข้างบน ก็ต้องเป็นตาม id ของ input นั้น ๆ